src/main/java/pl/ergohestia/ehj1/ivesta/ui/DriverMenu.java

```diff
@@ -0,0 +1,145 @@
+package pl.ergohestia.ehj1.ivesta.ui;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.ergohestia.ehj1.ivesta.model.DriverDto;
+import pl.ergohestia.ehj1.ivesta.model.Menu;
+import pl.ergohestia.ehj1.ivesta.services.DriverService;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+@Slf4j
+public class DriverMenu {
+
+    private Logger SYSOUT = LoggerFactory.getLogger("SYSOUT");
+    private InputScannerProvider in = new InputScannerProvider();
+
+    DriverService driverService = new DriverService();
+
+    public void runDriverMenu() {
+        serviceDriverMenu();
+    }
```

Comment:
do poprzedniego komentarza miałem na myśli tutaj coś takeigo

MenuSerivce menuService ;
    public void runDriverMenu(MenuService menuService){
        this.menuService = menuService;
        serviceVehicleMenu();
        menuService.menu(); // jeżeli coś wybierzesz w vehicle menu to po skończeniu to się wykona i wóci do głównego menu
        
    }
    
    a dalej tak jak pisałem żeby w punkcie 5 podmienić

Reply:
j.w.